app/views/teams/_sub_teams.html.erb

```diff
@@ -19,9 +19,8 @@
     <% end %>
   </div>
 <% end %>
-
```

Comment:
Do not remove these blank lines. They are kept for idnentifying the blocks

Reply:
this is happening on auto format while saving

Reply:
Added the Blank line